spec/lib/pdf_filler/md502_pdf_spec.rb

```diff
@@ -440,5 +439,21 @@
         expect(pdf_fields["Enter 13"].to_i).to eq 0
       end
     end
+
+    describe "Line 15" do
```

Comment:
minor thing but there's a section above `context "subtractions" do` that you could move these blocks into